CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+<a name="0.24.0"></a>
+# [0.24.0](https://github.com/NativeScript/nativescript-dev-webpack/compare/0.21.0...0.24.0) (2019-05-29)
```

Comment:
Resolved by updating the change log manually. We are not sure why the conventional-changelog tool does not find the 0.22.0 tag when triggering `npm --no-git-tag-version version 0.24.0 --allow-same-version -m "release: cut the %s release"` , at the moment that tool does not allow us to manually set the comparison range and automatically uses the latest tag but in our case it find 0.21.0 rather than 0.22.0